plugins/jobs/src/js/components/JobsOverviewTable.js

```diff
@@ -238,7 +239,11 @@ export default class JobsOverviewTable extends React.Component {
     const nodes = [];
     const statusNode = <span className={statusClasses}>{status}</span>;
 
-    if (lastFailureAt == null && lastSuccessAt == null) {
+    if (
+      (lastFailureAt == null && lastSuccessAt == null) ||
+      (lastFailureAt != null && !DateUtil.isValidDate(lastFailureAt)) ||
+      (lastSuccessAt != null && !DateUtil.isValidDate(lastSuccessAt))
```

Comment:
if we'd make `isValidDate` aware of `null`, could we simplify this to the following?

```typescript
if (!DateUtil.isValidDate(lastFailureAt) || !DateUtil.isValidDate(lastSuccessAt)){
  return statusNode;
}
```

Reply:
Good idea, `null` is not a valid date anyway.